clicommand/artifact_search.go

```diff
@@ -162,6 +164,7 @@ var ArtifactSearchCommand = cli.Command{
 				"%j", artifact.JobID,
 				"%s", strconv.FormatInt(artifact.FileSize, 10),
 				"%S", artifact.Sha1Sum,
+				"%t", artifact.Sha256Sum,
```

Comment:
Personally I'd go uppercase to make it nearer to `%S` and further from `%s` but 🤷 
```suggestion
				"%T", artifact.Sha256Sum,
```